### ISSUE TITLE
Cannot  generate items by "Update Database"

### ISSUE THREAD

**Issue by denven** [cboxdoerfer/fsearch#191]
Unbutun 20.04, installed from PPA

It says 0 items in right bottom corner, and it still has no index even when clicking "Update Database" several times.
So fast search cannot work.

**Comment by Lamecarlate** [comment]
Did you go to Settings > Database (not sure of the exact term, I'm retrotranslating from my french installation :stuck_out_tongue:  ) and include folders?

**Comment by denven** [comment]
> Did you go to Settings > Database (not sure of the exact term, I'm retrotranslating from my french installation  ) and include folders?

Hi, Lamecarlate, thanks for reply, I got you and get the answer, that's my bad.

I choose 'File' -> 'Update Database' and there is no action happened, no database index is built. That's what I did before.

From what you mentioned, I checked 'Edit' menu's "Preference ", there is a tab "Database" which should include searching paths used for searching and building database index.

I thought fsearch has a full disk coverage search by default.

This "issue" can be closed.
